packages/core/src/modules/proofs/protocol/v1/messages/V1RequestPresentationMessage.ts

```diff
@@ -1,23 +1,16 @@
-import type { ProofAttachmentFormat } from '../../../formats/models/ProofAttachmentFormat'
 import type { IndyProofRequest } from 'indy-sdk'
 
 import { Expose, Type } from 'class-transformer'
 import { IsArray, IsString, ValidateNested, IsOptional, IsInstance } from 'class-validator'
 
 import { AgentMessage } from '../../../../../agent/AgentMessage'
 import { Attachment } from '../../../../../decorators/attachment/Attachment'
-import { AriesFrameworkError } from '../../../../../error/AriesFrameworkError'
-import { JsonTransformer } from '../../../../../utils/JsonTransformer'
 import { IsValidMessageType, parseMessageType } from '../../../../../utils/messageType'
-import { V2_INDY_PRESENTATION_REQUEST } from '../../../formats/ProofFormatConstants'
-import { ProofRequest } from '../../../formats/indy/models/ProofRequest'
-import { ProofFormatSpec } from '../../../models/ProofFormatSpec'
 
-export interface RequestPresentationOptions {
+export interface V1RequestPresentationMessageOptions {
   id?: string
   comment?: string
-  parentThreadId?: string
-  requestPresentationAttachments: Attachment[]
+  requestAttachments: Attachment[]
```

Comment:
Same as above

Reply:
I can't find the comment this is related to?

Reply:
Nevermind! It was a question about pthid but deleted because I found the answer in another change